.gitignore

```diff
@@ -53,8 +53,6 @@ spec/integration/data/gold
 /spec/manual/public/exceljs.js
 /spec/manual/public/exceljs.min.js
 
-package-lock.json
```

Comment:
`patch-package` requires `package-lock` to work. I can extract this to a separate PR if necessary